subgraph-gnosis-chain.yaml

```
@@ -208,6 +208,28 @@ dataSources:
       eventHandlers:
         - event: PairCreated(indexed address,indexed address,address,uint256)
           handler: handleNewPair
+  - kind: ethereum/contract
+    name: ds-balancer-v1-factory
+    network: xdai
+    source:
+      # symmetric's factory address, a balancer-v1 clone
+      address: "0x9B4214FD41cD24347A25122AC7bb6B479BED72Ac"
+      abi: Factory
+      startBlock: 13108160
```

```suggestion
      startBlock: 16465037
```

https://blockscout.com/xdai/mainnet/tx/0x20020bccdaf2361e40834f6530b10fbb142ddfe1339519329dc9e5ba7a3a35e3